### DISCUSSION TITLE
CORe50, 10 classes

### DISCUSSION THREAD

**Discussion by DraBard** [ContinualAI/avalanche#1453] [Q&A] [answered]
Can some1 explain to me why `core50 = CORe50(scenario="nc", mini=True, object_lvl=False)` (when the dataset has 10 classes) the experiences have the following classes:

exp 0: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
exp 1:  [5, 6, 7, 8, 9]
exp 2: [0, 1, 2, 3, 4]
exp 3:  [5, 6, 7, 8, 9]
exp 4: [0, 1, 2, 3, 4]
etc ... 

Isn't the whole idea of new classes scenario, that each experience has a new class introduced?

Is there an easy way to get to the configuration where each experience has one class only? This is what I need ...

For `object_lvl=False` it seems to work as expected ...

**Comment by vlomonaco** [comment, marked as answer]
Hi @Bard2803, you are right, this may be a possible bug!

> Is there an easy way to get to the configuration where each experience has one class only? This is what I need ...

Sure, we recommend using the benchmark generators for this: https://avalanche.continualai.org/from-zero-to-hero-tutorial/03_benchmarks#benchmarks-generators
